app/common_types.py

```diff
@@ -46,3 +45,36 @@ class GameMode(IntEnum):
     TAIKO = 1
     CATCH = 2
     MANIA = 3
+
+
+class RelaxMode(IntEnum):
+    VANILLA = 0
+    RELAX = 1
+    AUTOPILOT = 2
+
+
+class Mode(IntEnum):
```

Comment:
maybe AkatsukiMode? shiiit it sucks that we have both types of modes now, i should have probably key'd by (mode, rx) in stats db